test/build.test.js

```diff
@@ -83,7 +83,9 @@ describe('Build Test', () => {
 
     await builder.run();
 
-    await assertZipEntries(path.resolve(testRoot, 'dist', 'simple-package', 'simple-name@1.45.0.zip'), [
+    const zipPath = path.resolve(testRoot, 'dist', 'simple-package', 'simple-name@1.45.0.zip');
+    assert.ok(fse.existsSync(zipPath), `Build did not produce zip file: ${zipPath}`);
```

Comment:
FYI This is unrelated to the PR, but when unit tests failed here, it didn't show the stack trace but only the file not found error. This extra assertion makes that nicer.